src/Microsoft.Azure.WebJobs.Host/Bindings/OpenType.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    /// <summary>
+    /// Placeholder to use with converter manager for describing open types. 
+    /// The type can be filtering via a IsValid() method. 
```

Comment:
This summary text isn't really clear to me. I think very clear documentation for this pattern if it is convention based.